lab_2/filtration.py

```diff
@@ -0,0 +1,208 @@
+from tqdm import tqdm
+import numpy as np
+from skimage import img_as_ubyte, img_as_float
+from skimage.color import rgb2gray
+# from skimage import io
+
+
+def wiener(files, progress_bar, progress_label, root):
+    from skimage import restoration
+    progress_step = 100 / len(files)
+    progress_bar['value'] = 0
+    progress_label.config(text="0")
+    root.update_idletasks()
+    result_array = []
+    psf = np.ones((5, 5)) / 25
+    for i in tqdm(range(0, len(files)), desc="Фильтрация: "):
+        img = img_as_float(rgb2gray(files[i]))
+        restored_img, _ = restoration.unsupervised_wiener(img, psf)
+        restored_img = img_as_ubyte(
```

Comment:
Также можно вынести вообще всю логику в отдельную функцию:

```suggestion
        def restore_img(image):
            normalized = normalize_image(image)
            return img_as_ubyte(normalized)
```